packages/api/src/external/carequality/command/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -86,24 +87,35 @@ function getUrls(contained: Contained): XCUrls | undefined {
     }
   });
 
+  const urls: XCUrls = {};
   const urlXCPD = endpointMap[XCPD_STRING];
+  const urlDQ = endpointMap[XCA_DQ_STRING];
+  const urlDR = endpointMap[XCA_DR_STRING];
 
-  if (!urlXCPD) return;
```

Comment:
Now that we don't return `undefined` anymore, let's update the function's return type?